main/export-list.js

```diff
@@ -22,6 +23,7 @@ const Export = require('./export');
 
 const ffmpegPath = util.fixPathForAsarUnpack(ffmpeg.path);
 const showSaveDialog = pify(dialog.showSaveDialog, {errorFirst: false});
+let lastSavedDirectory = null;
```

Comment:
There's no reason to initialize with `null`.
```suggestion
let lastSavedDirectory;
```